tests/unit/dataset/dataset_test.py

```diff
@@ -773,6 +783,7 @@ def test_load_no_files_raises_exception(gaze_dataset_configuration):
         dataset.scan()
 
 
+@pytest.mark.filterwarnings('ignore:.*load_function.*:UserWarning')
```

Comment:
warning raised as we do not provide any public gaze sample load function for mat files yet